### DISCUSSION TITLE
Is there something that can be added to the config to omit certain search suggestions?

### DISCUSSION THREAD

**Discussion by linlinxza** [searxng/searxng#3840] [Q&A] [answered]
For example, if I type in garbage like "facebook", I was wondering if there was a way to not make something that show up in the search suggestions.

I want to act like Facebook (and other social media) doesn't exist. :)

**Comment by return42** [comment, marked as answer]
You can activate the [Hostnames plugin](https://docs.searxng.org/src/searx.plugins.hostnames.html#hostnames-plugin) and in the config ignore results linking to facebook:

```yaml
hostnames:
  remove:
    - '(.*\.)?facebook.com$'
```

**Reply by linlinxza** [reply]
> You can activate the [Hostnames plugin](https://docs.searxng.org/src/searx.plugins.hostnames.html#hostnames-plugin) and in the config ignore results linking to facebook:
> 
> ```yaml
> hostnames:
>   remove:
>     - '(.*\.)?facebook.com$'
> ```

I've already done this. 

I mean search suggestions that appear based on keywords: 

![image](https://github.com/user-attachments/assets/a6187422-d9ef-4aba-81c6-1f211706d9ab)

**Reply by return42** [reply]
nope, that is not possible

**Reply by linlinxza** [reply]
Shoot. I wish it was possible to filter search suggestions.
